package/cloudshell/iac/terraform/services/sandox_data.py

```diff
@@ -21,7 +21,7 @@ def _get_tf_uuid(self) -> str:
         else:
             # Create new uuid and set it on the the attribute
             new_uuid = uuid.uuid4().hex
-            self._driver_helper_obj.tf_service.uuid = new_uuid
+            # self._driver_helper_obj.tf_service.uuid = new_uuid
```

Comment:
I think we need to save the UUID in tf_service because its used later. @qualidan is it correct?

Reply:
we are saving it in an attribute further down